controller.py

```diff
@@ -69,10 +69,10 @@ def edit_criteria_select(self, k=None):
         for index,score in enumerate(self.current_criterion.scores):
                 print(f"{index}: {score}")
         
-        self.current_score = self.current_criterion.scores[int(k)]
 
     def edit_score_confirm(self,k=None):
         """Confirm changes to score"""
+        self.current_score = self.current_criterion.scores[int(k)]
```

Comment:
Do we need to store the score at all now that we've removed the confirmation step?

Reply:
Yes, go ahead.